mcm/json_layer/request.py

```diff
@@ -1872,9 +1872,46 @@ def get_stats(self, forced=False):
                                                   page=0,
                                                   limit=-1,
                                                   options={'key': prepid})
+        # easier to access this way below
+        stats_workflows_dict = dict([(stats_workflow.get('RequestName'), stats_workflow) for stats_workflow in stats_workflows])
```

Comment:
This was already available in Python 2.7:
```suggestion
        stats_workflows_dict = {stats_workflow.get('RequestName'): stats_workflow for stats_workflow in stats_workflows}
```